packages/components-css/logo/index.scss

```diff
@@ -0,0 +1,68 @@
+// LOGO STYLING
+.frameless-logo__wrapper {
+  align-items: center;
+  container: logowrapper / inline-size;
+  display: flex;
+  flex-basis: min-content;
+  flex-direction: row;
+  justify-content: center;
+}
+
+.frameless-logo__text {
+  display: inline-block;
+  font-family: var(--frameless-logo-text-font-family);
+  font-size: var(--frameless-logo-text-font-size-large);
```

Comment:
deze moet volgens de naming convention `--frameless-logo-text-large-font-size` zijn. Ik zou voorstellen om dit gewoon zonder `-large` te doen.

Reply:
Zo had ik het eerst, maar omdat ik van de 2(r)em/3(r)em afwilde heb ik font-size in de json opgesplitst in small en large (dit zijn de 2 varianten in het logo). Daar ziet de component opbouw er nu zo uit:

<img width="647" alt="logo-component-token-opbouw-font-size" src="https://github.com/frameless/website/assets/112861180/0938cdc1-6451-485d-97e6-d7eea99aea57">

* Het is de bedoeling dat ik de volgorde uit de json volg toch? Of is het andersom en is er een andere naming convention die ik ook op de json hoor toe te passen? 🤔 Is er een plek waar ik deze conventions terug kan vinden?
* Heb je liever small en medium als er twee varianten zijn? (dit voelde voor mij niet logisch omdat er geen derde is, dus waar zou medium dan de middelste van zijn.. 🙃)
* Zijn er opties waarbij het wel logisch is om alleen --frameless-logo-text-font-size in de tokens te zetten en dan in de scss te manipuleren tot de juiste size? (Maybe calc? - ik weet van het bestaan maar ben niet zo'n wiskunde held dus ben hier een beetje van weg gebleven tot nu toe + is het dan duidelijk welke size erbij hoort?)